mitiq/zne/tests/test_zne.py

```diff
@@ -560,3 +573,74 @@ def qs_noisy_simulation(
 
     mitigated = execute_with_zne(circuit, qs_noisy_simulation)
     assert abs(mitigated) < 1000
+
+
+@pytest.mark.parametrize(
+    "noise_scaling_method",
+    [fold_gates_at_random, insert_id_layers, fold_global, fold_all],
+)
+@pytest.mark.parametrize(
+    "extrapolation_factory", [RichardsonFactory, LinearFactory]
+)
+@pytest.mark.parametrize(
+    "to_frontend",
+    [None, to_qiskit, to_braket, to_pennylane, to_pyquil, to_qibo],
```

Comment:
```suggestion
    [convert_to_mitiq, to_qiskit, to_braket, to_pennylane, to_pyquil, to_qibo],
```

I think we can use this instead to leave the circuit as a `cirq.Circuit` object instead of doing the check below.

Reply:
That makes it a bit cleaner indeed!

Reply:
Actually, convert_to_mitiq() returns a tuple (the circuit and a string). So you still need one check right (if to_frontend = convert_to_mitiq)?